tests/Metadata/MetaDataBaseTest.php

```diff
@@ -285,28 +285,6 @@ public function testInvalidField(string $expectedField, string $expectedType) :
         static::callCreateFromJson($json);
     }
 
-    /**
-     * Change a nested array element.
-     *
-     * @param array $keys
-     *   Ordered keys to the value to set.
-     * @param array $data
-     *   The array to modify.
-     * @param mixed $newValue
-     *   The new value to set.
-     *
-     * @return void
-     */
-    protected function nestedChange(array $keys, array &$data, $newValue) : void
-    {
-        $key = array_shift($keys);
-        if ($keys) {
-            $this->nestedChange($keys, $data[$key], $newValue);
-        } else {
-            $data[$key] = $newValue;
-        }
-    }
-
```

Comment:
No change this method just moved to existing `UtilsTrait` so it can be used in `TestRepo`